src/web/app/pages-instructor/instructor-courses-page/instructor-courses-page.component.ts

```diff
@@ -319,13 +319,20 @@ export class InstructorCoursesPageComponent implements OnInit {
     .subscribe(() => {
       // Wrap in a Promise to wait for all feedback sessions to be copied
       const promise: Promise<void> = new Promise<void>((resolve: () => void) => {
+        if (result.selectedFeedbackSessionList.size < 1) {
```

Comment:
Small nit, might be clearer this way
```suggestion
        if (result.selectedFeedbackSessionList.size === 0) {
```

Reply:
I agree on this! Initially, wanted to use `0` but forgot what came to me to pick `< 1`. Thank you!